nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamProcessorDescriptor.java

```diff
@@ -224,9 +232,15 @@ protected ComputationPolicy getComputationPolicy(PolicyDescriptor policyDescript
                                              .batchPolicy(policyDescriptor.batchCapacity,
                                                      policyDescriptor.batchThreshold)
                                              .continueOnFailure(policyDescriptor.continueOnFailure)
+                                             .skipFirstFailures(getSkipFirstFailures(policyDescriptor))
                                              .build();
     }
 
+    protected int getSkipFirstFailures(PolicyDescriptor policyDescriptor) {
+        return Integer.parseInt(Framework.getProperty(RECOVERY_SKIP_FIRST_FAILURES_OPTION,
```

Comment:
Shouldn't be the opposite? Descriptor should states a value and if none we retrieve from a more global configuration properties.
Furthermore you could use `ConfigurationService` instead of `Framework#getProperty,` or this kind of properties are system enough to be in `nuxeo.conf`?

Reply:
This option is used for exceptional manual recovery, we don't want to change the policy of the computation in the descriptor neither we want to deploy a new contribution that make the usage complex and error-prone.
To use the recovery mode you add the option to the nuxeo.conf then restart and remove the option.